From d2dbd60894a2ba61859f4424368cbcb82eb99ab5 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Wed, 16 May 2018 10:04:01 +0800
Subject: [PATCH] PM / AVS: rockchip-io: Define the px30-specific supply names

Also define the px30-specific supply names as can be seen in that
file for the other socs.

vccio-oscgpi ... aka with a "-" instead of "_".

Change-Id: I465e18123211b567f14d10e9705d9194fdfbd33b
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 .../bindings/power/rockchip-io-domain.txt           | 13 +++++++++++++
 drivers/power/avs/rockchip-io-domain.c              |  2 +-
 2 files changed, 14 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/power/rockchip-io-domain.txt b/Documentation/devicetree/bindings/power/rockchip-io-domain.txt
index 5c162a7ac85a..eb0714b019f8 100644
--- a/Documentation/devicetree/bindings/power/rockchip-io-domain.txt
+++ b/Documentation/devicetree/bindings/power/rockchip-io-domain.txt
@@ -55,6 +55,19 @@ a phandle the relevant regulator.  All specified supplies must be able
 to report their voltage.  The IO Voltage Domain for any non-specified
 supplies will be not be touched.
 
+Possible supplies for PX30:
+- vccio6-supply: The supply connected to VCCIO6.
+- vccio1-supply: The supply connected to VCCIO1.
+- vccio2-supply: The supply connected to VCCIO2.
+- vccio3-supply: The supply connected to VCCIO3.
+- vccio4-supply: The supply connected to VCCIO4.
+- vccio5-supply: The supply connected to VCCIO5.
+- vccio-oscgpi-supply: The supply connected to VCCIO_OSCGPI.
+
+Possible supplies for PX30 pmu-domains:
+- pmuio1-supply: The supply connected to PMUIO1.
+- pmuio2-supply: The supply connected to PMUIO2.
+
 Possible supplies for rk3188:
 - ap0-supply:    The supply connected to AP0_VCC.
 - ap1-supply:    The supply connected to AP1_VCC.
diff --git a/drivers/power/avs/rockchip-io-domain.c b/drivers/power/avs/rockchip-io-domain.c
index 5b6bea5236d1..fc2f701137c4 100644
--- a/drivers/power/avs/rockchip-io-domain.c
+++ b/drivers/power/avs/rockchip-io-domain.c
@@ -306,7 +306,7 @@ static const struct rockchip_iodomain_soc_data soc_data_px30 = {
 		"vccio3",
 		"vccio4",
 		"vccio5",
-		"vccio_oscgpi",
+		"vccio-oscgpi",
 	},
 	.init = px30_iodomain_init,
 };
-- 
2.35.3

